tests/Mock/AdapterCanHandlerMock.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * This file is part of the prooph/service-bus.
+ * (c) 2014-2016 prooph software GmbH <contact@prooph.de>
+ * (c) 2015-2016 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ProophTest\EventStore\Mock;
+
+use Prooph\EventStore\Adapter\Feature\CanHandleTransaction;
+use Prooph\EventStore\Adapter\InMemoryAdapter;
+
+/**
+ * Class AdapterCanHandlerMock
+ *
+ * @package ProophTest\EventStore
+ * @author Jefersson Nathan <malukenho@phpse.net>
+ */
+final class AdapterCanHandlerMock extends InMemoryAdapter implements CanHandleTransaction
```

Comment:
I would suggest renaming that class to TransactionalInMemoryAdapterMock, so the naming is a little more clear.